integreat_cms/static/src/js/pois/poi-actions.ts

```diff
@@ -35,8 +45,51 @@ window.addEventListener("load", () => {
       latitude.dataset.initial = latitude.value;
     });
   }
+
+  document.getElementById("large_map_preview_button")?.addEventListener("click",  ()=> showPreviewLarge(largeMap));
 });
 
+function showPreviewLarge(container:HTMLElement){
+  showPreview(container);
+  
+  document.getElementById("map_preview_overlay")?.classList.remove("hidden");
+  document.getElementById("btn-close-preview")?.addEventListener("click", closePreview);
+}
+
+function showPreview(container:HTMLElement){
+  var lng = (document.getElementById("id_longitude") as HTMLInputElement).value;
+  var lat = (document.getElementById("id_latitude") as HTMLInputElement).value;
+
+  if (lng && lat){
+    container.classList.remove("hidden");
+    document.getElementById("large_map_preview_button")?.classList.remove("hidden");
+    document.getElementById("no_coordinates_alert")?.classList.add("hidden");
+
+    var map = new maplibregl.Map({
+      container: container, //'map',
+      style: 'https://maps.tuerantuer.org/styles/integreat/style.json',
+      center: [parseFloat(lng), parseFloat(lat)], // starting position [lng, lat]
+      zoom: 12 // starting zoom
```

Comment:
```suggestion
      zoom: 15 // starting zoom
```
I'd go with a slightly enlarged zoom factor